site/en/setup.md

```diff
@@ -11,13 +11,24 @@ OCR-D's software is a modular collection of many projects (called _modules_)
 with many tools per module (called _processors_) that you can combine freely
 to achieve the workflow best suited for OCRing your content.
 
+## System requirements
+
+Minimum system requirements
+* 8 GB RAM (more recommended)
+* min. 100 GB free disk space for local installation 
+  - (min/max similar to Docker?)
+* Linux OS (we recommend Ubuntu 18.04 for native installation)
```

Comment:
Should we mention Windows 10 with WSL here, too? I think that would help a lot of people doing the first steps.

Reply:
For some reason I thought we already had a link to the further setup tutorials in the Wiki. But you are right, I will add it

Reply:
The link is still missing, right?

Reply:
just added it